plugins/LanguagesManager/API.php

```diff
@@ -275,6 +275,45 @@ public function setLanguageForUser($login, $languageCode)
         return true;
     }
 
+    /**
+     * Returns whether the user uses 12 hour clock
+     *
+     * @param string $login
+     * @return string
+     */
+    public function uses12HourClockForUser($login)
```

Comment:
Just a random question, are there other time formats apart from `24 hours` and `12 hours am/pm`? If so, it should maybe return not a boolean but maybe a string or so. Not sure if there are possible other time formats